### DISCUSSION TITLE
Installing in ArchLinux

### DISCUSSION THREAD

**Discussion by milantimotijevic** [KhronosGroup/KTX-Software#649] [General]
Hi everyone,

I need to install this inside an AWS Lambda NodeJS container. Basically, it's this one amazon/aws-lambda-nodejs:12

This is the distro:
`NAME="Amazon Linux"
VERSION="2"
ID="amzn"
ID_LIKE="centos rhel fedora"
VERSION_ID="2"
PRETTY_NAME="Amazon Linux 2"
ANSI_COLOR="0;33"
CPE_NAME="cpe:2.3:o:amazon:amazon_linux:2"
HOME_URL="https://amazonlinux.com/"
VARIANT_ID="202209281805-2.0.949.0"
`
I assumed I needed to install the .rpm package and I tried running
`yum install KTX-Software-4.1.0-rc3-Linux-x86_64.rpm`
However, I ended up getting this error message:
`--> Finished Dependency Resolution
Error: Package: ktx-software-4.1.0-1.x86_64 (/KTX-Software-4.1.0-rc3-Linux-x86_64)
           Requires: libktx.so.4()(64bit)
Error: Package: ktx-software-4.1.0-1.x86_64 (/KTX-Software-4.1.0-rc3-Linux-x86_64)
           Requires: libm.so.6(GLIBC_2.27)(64bit)
Error: Package: ktx-software-4.1.0-1.x86_64 (/KTX-Software-4.1.0-rc3-Linux-x86_64)
           Requires: libstdc++.so.6(GLIBCXX_3.4.26)(64bit)
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
`
Am I doing something wrong, or is this specific binary broken?

The debian one works great, but I cannot install it on this distro.

**Comment by MarkCallow** [comment]
You asked this before in #612. Let's continue discussion there.
